parsec/api/data/manifest.py

```diff
@@ -71,6 +71,12 @@ def make_obj(self, data: Dict[str, Any]) -> "WorkspaceEntry":
 T = TypeVar("T")
 BaseAPISignedDataTypeVar = TypeVar("BaseAPISignedDataTypeVar", bound="BaseAPISignedData")
 BaseManifestTypeVar = TypeVar("BaseManifestTypeVar", bound="BaseManifest")
+AnyManifest = Union[
```

Comment:
`AnyManifest` should be defined in the .pyi

Reply:
will be done in #2865